.eslintrc

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
It is, but no semicolon usage makes for inconsistent reading in the earlier Node versions. You got away with in the previous versions because Babel added them in accordingly